roles/confluent.kafka_broker/templates/server.properties.j2

```diff
@@ -1,114 +1,4 @@
 # Maintained by Ansible
-zookeeper.connect={% for host in groups['zookeeper'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{zookeeper.properties.clientPort}}{% endfor %}
-
-log.dirs={% for logdir in kafka_broker.datadir %}{% if loop.index > 1%},{% endif %}{{ logdir }}{% endfor %}
-
-broker.id={{ broker_id | default(groups.kafka_broker.index(inventory_hostname) + 1)}}
-
```

Comment:
Might be worth leaving a comment that confirms where the properties are merged together and then pull from.  Can be a a bit confusing to see an almost empty properties file.

Reply:
what specifically should the comment say? Its clear in the code that all properties live in the kafka_broker_final_properties dictionary. If you want to know whats in that dictionary you should look where its defined

Reply:
I am suggesting saying that kafka_broker_final_properties is built in X location.

Reply:
i dont see how this var is different than all the other vars in the variables and handlers role. kafka_broker_ssl_enabled for example, prior to this pr we didn't document "kafka_broker_ssl_enabled is defined in roles/confluent.variables_handlers/defaults/main.yml"